app/controllers/admin/lead_providers/lead_provider_users_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Admin
+  module LeadProviders
+    class LeadProviderUsersController < Admin::BaseController
+      skip_after_action :verify_policy_scoped
+      before_action :load_lead_provider_user
+
+      def edit
+        authorize @lead_provider_user
+      end
+
+      def update
+        authorize @lead_provider_user
+
+        if @lead_provider_user.update(permitted_attributes(@lead_provider_user))
```

Comment:
Whoa, I was expecting it to require playing with our policies, but it seems someone had enough foresight to put `permitted_attributes` for the user in :O